here_for_change/municipalities/templates/municipalities/find_my_ward_councillor.html

```diff
@@ -0,0 +1,177 @@
+                        <span class="hidden md:block pr-2">OR</span>
+                        <button id="embed-location-btn"
+                            class="flex items-center justify-center w-full h-9  md:w-[180px] text-sm rounded-md text-white whitespace-nowrap text-center font-medium bg-[#7f00ff] transition-[background-color] duration-200 ease hover:bg-[#4b0097]">Use
+                            your location
+                        </button>
+                        <form id="redirect-to-closest-location" action="{% url 'redirect-to-closest-ward' %}" method="get" style="display: none;">
+                            <input type="number" name="longitude">
+                            <input type="number" name="latitude">
+                            <input type="text" name="url">
+                        </form>
+                    </div>
+                    <!-- loader -->
+                    <div id="use-location-loader"
+                        class="hidden absolute rounded-md bg-hfc-green-700 h-full w-full flex items-center">
+                        <i class="fa-solid fa-spinner text-white text-base animate-spin px-3"></i>
+                        <p class="text-white text-sm">Finding your ward councillor</p>
+                    </div>
+                </div>
+            </div>
+            <p class="text-white text-xs opacity-60 mt-[9px]">We will never share any of your information with any third
+                parties</p>
+            <div class="border-white opacity-10 w-full border-t my-[21px] md:my-[31px]"></div>
+            <div class="gap-y-4 flex flex-col sm:flex-row items-center justify-between">
+                <div class="flex flex-col">
+                    <a class="text-lg inline-block benfritz text-white lg:text-[21px] relative" href="#">
+                        <span>HereForChange</span>
+                        <div
+                            class="w-8 h-8 lg:w-[35px] lg:h-[35px] absolute -top-[17px] -right-[22px] lg:-right-[23px] flex items-center justify-center rotate-[-17deg]">
+                            <img class="w-8 h-8 lg:w-[35px] lg:h-[35px]" src="{% static 'assets/SA-flag.png' %}" />
+                        </div>
+                    </a>
+                    <p class="text-white text-[11px] opacity-60 -mt-1">
+                        A tool created by <a class="underline" href="https://openup.org.za" target="_blank" rel="noopener noreferrer">OpenUp</a> 
+                    </p>
+                </div>
+                <div class="flex items-center gap-x-2">
+                    <p class="text-white text-sm opacity-60">Share this tool:</p>
+                    <a class="voteTooltip w-10 h-10 bg-hfc-green-700 flex items-center justify-center rounded-lg" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fhere-for-cha-home-mvp-5lckwdmi.herokuapp.com%2F&amp" target="_blank" rel="noopener noreferrer">
```

Comment:
`here-for-cha-home-mvp-5lckwdmi.herokuapp.com` should not be hard coded in @Hazmid. Couple other occurrences of this. @cyber237 can help if you need the host name.